content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -22,6 +22,34 @@ Most of the blog's content is about things happening in the core project, but we
 
 Anyone can write a blog post and submit it for review.
 
+### Submit a Post
+
+Blog posts should not be commercial in nature and should consist of original content that applies broadly to the Kubernetes community.
+Appropriate blog content includes:
+
+- New Kubernetes capabilities
+- Kubernetes projects updates
```

Comment:
Hi @chrisnegus . I read through the entire page. I did not realize that part of this new section
duplicates some of the content found in the `Guidelines and expectations` section in the page. Does it make sense to
push the "Appropriate and Unsuitable content" into the `Guidelines` section?
I like the steps that describe how to create a blog post.